# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/indexedDB/PBackgroundIDBCursor.ipdl
# Commit: d2772a7482ed
# Full Hash: d2772a7482ed9394a2c30acc6d03caa36317e75f
# Author: Simon Giesecke <sgiesecke@mozilla.com>
# Date: 2019-09-30 16:21:37
# Regressor Bug: 1168606
# File Overlap Count: 1
# Description:
#   Bug 1168606 - Allow current key to be provided with PBackgroundIDBCursor::Continue. r=ttung,asuth
#   
#   The key is assumed to be unset for now.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D43251
# ==============================================================================

diff -r 340dfa69aed8 -r d2772a7482ed dom/indexedDB/PBackgroundIDBCursor.ipdl
--- a/dom/indexedDB/PBackgroundIDBCursor.ipdl	Fri Sep 27 10:11:45 2019 +0000
+++ b/dom/indexedDB/PBackgroundIDBCursor.ipdl	Fri Sep 27 10:11:41 2019 +0000
@@ -90,7 +90,7 @@
 parent:
   async DeleteMe();
 
-  async Continue(CursorRequestParams params);
+  async Continue(CursorRequestParams params, Key currentKey);
 
 child:
   async __delete__();
